src/wcmUSB.c

```diff
@@ -2085,14 +2085,15 @@ static int usbProbeKeys(WacomDevicePtr priv)
 TEST_CASE(test_mod_buttons)
 {
 	WacomCommonRec common = {0};
-	for (size_t i = 0; i < sizeof(int) * 8; i++)
+	for (size_t i = 0; i < sizeof(int) * 8 - 1; i++)
 	{
 		unsigned int buttons = mod_buttons(&common, 0, i, 1);
 		assert(buttons == (1u << i));
 		buttons = mod_buttons(&common, 0, i, 0);
 		assert(buttons == 0);
 	}
 
+	assert(mod_buttons(&common, 0, sizeof(int) * 8 - 1, 1) == 0);
 	assert(mod_buttons(&common, 0, sizeof(int) * 8, 1) == 0);
```

Comment:
This line needs to be removed, right?

Reply:
doesn't need to, it'll still test a boundary condition even though it's no longer *the* boundary condition. doesn't hurt to leave it in, we have the µs to spare :)